WordPress/gutenberg
# Remove generated gutenberg.pot file from repository

Closes #604 

This pull request seeks to ignore the `languages/gutenberg.pot` in the repository, instead moving string extraction to a standalone npm script `gettext-strings`.

__Testing instructions:__

Verify that the `languages/gutenberg.pot` file is not included in the repository, but that it can still be generated by running `npm run gettext-strings`.
